examples/run_language_modeling.py

```diff
@@ -476,7 +550,9 @@ def collate(examples: List[torch.Tensor]):
         for key in sorted(result.keys()):
             logger.info("  %s = %s", key, str(result[key]))
             writer.write("%s = %s\n" % (key, str(result[key])))
-
+    if hasattr(eval_dataset, "close"):
+        # For lazy loaders, clean up file handles.
+        eval_dataset.close()
```

Comment:
Is this necessary? Where is an unclosed file handle?

Reply:
Legacy of how I was doing it before switching to linecache. Not required anymore.